builder/virtualbox/ovf/config.go

```diff
@@ -87,15 +90,24 @@ func NewConfig(raws ...interface{}) (*Config, []string, error) {
 	errs = packer.MultiErrorAppend(errs, c.VBoxManagePostConfig.Prepare(&c.ctx)...)
 	errs = packer.MultiErrorAppend(errs, c.VBoxVersionConfig.Prepare(&c.ctx)...)
 
-	if c.SourcePath == "" {
+	// Get downloadable URL
+	downloadableURL, err := common.DownloadableURL(c.SourcePath)
+	notURL := !strings.HasPrefix(downloadableURL, "http://") && !strings.HasPrefix(downloadableURL, "https://")
```

Comment:
Thanks again for all the feedback. I completely agree. 

The reason I did this was because I expected [DownloadableURL](https://github.com/mitchellh/packer/blob/d310de8cf5871354bd1d12996a87020803d373ea/common/config.go#L77-L89) to return an error when `/i/dont/exist` was passed in, but it didn't, so I did my own stat, which is something I'd only want to do if the `source_path` was not a URL.